docs/src/SUMMARY.md

```diff
@@ -4,6 +4,7 @@
 
 # Tutorials
 
+- [Entry-level Model Checker Tutorial](./tutorials/entry-tutorial.md)
```

Comment:
Given the size of this tutorial, would it be useful to break it into subsections, like we do with many of the other sections in the book? Or do you think it's better to keep it "flat" is it is?

Reply:
Right. I have merged these sections.

Reply:
Yea, that's what I was thinking too. We can add this preprocessor to get TOCs for each page: https://github.com/JorelAli/mdBook-pagetoc

I'll open a separate issue for that.